assembl/tests/views_tests/test_graphql.py

```diff
@@ -1274,21 +1338,22 @@ def test_get_proposals(graphql_request, thematic_and_question, proposals15publis
                     u'hasPreviousPage': False,
                     u'hasNextPage': True
                 },
-                u'edges': [{u'node': {u'body': u'une proposition 14'}},
-                           {u'node': {u'body': u'une proposition 13'}},
-                           {u'node': {u'body': u'une proposition 12'}},
-                           {u'node': {u'body': u'une proposition 11'}},
-                           {u'node': {u'body': u'une proposition 10'}},
-                           {u'node': {u'body': u'une proposition 9'}},
-                           {u'node': {u'body': u'une proposition 8'}},
-                           {u'node': {u'body': u'une proposition 7'}},
+                u'edges': [{u'node': {u'body': u'une proposition 0'}},
+                           {u'node': {u'body': u'une proposition 1'}},
+                           {u'node': {u'body': u'une proposition 2'}},
+                           {u'node': {u'body': u'une proposition 3'}},
+                           {u'node': {u'body': u'une proposition 4'}},
+                           {u'node': {u'body': u'une proposition 5'}},
                            {u'node': {u'body': u'une proposition 6'}},
-                           {u'node': {u'body': u'une proposition 5'}}]
+                           {u'node': {u'body': u'une proposition 7'}},
+                           {u'node': {u'body': u'une proposition 8'}},
+                           {u'node': {u'body': u'une proposition 9'}}]
```

Comment:
Because of your `@freeze_time`, the creation_date of proposals created in the `proposals15published` fixture have the same value and thus the order here is reversed. The query normally return the proposals in descending order and we tested that here. A proper fix should be to force the creation_date of the proposal in the fixture, increment it by one minute for each created proposal for example.

Reply:
Corrected with `tick` option from freezegun (https://github.com/spulec/freezegun#tick-argument)